share/util/shr_pio_mod.F90

```diff
@@ -669,8 +669,10 @@ subroutine shr_pio_getiotypefromname(typename, iotype, defaulttype)
        iotype = pio_iotype_netcdf4p
     else if ( typename .eq. 'NETCDF4C') then
        iotype = pio_iotype_netcdf4c
+#ifdef PIO2
```

Comment:
@jayeshkrishna , this particular code may have been broken for a while, but I think this easy fix should be what we want.

Reply:
Yes, I agree